system.go

```diff
@@ -251,7 +251,9 @@ func getiOSVersion(uaPlatformGroup string) int {
 	if len(s) == 0 {
```

Comment:
Line 250 defines `s` as a match to the RegEx `(cpu|iphone) os \\d+`. Line 251 handles when there is no match, and exits as 0 (unknown). Any positive regex match would be 8+ char length and end in one or more digits so should not be able to panic, correct?